plugin-server/src/utils/db/cassandra/behavioural-counter.repository.ts

```diff
@@ -0,0 +1,109 @@
+import { Client as CassandraClient, types as CassandraTypes } from 'cassandra-driver'
+
+export interface BehavioralCounterRow {
+    team_id: number
+    filter_hash: string
+    person_id: string
+    date: string
+    count: number
+}
+
+export interface CounterUpdate {
+    teamId: number
+    filterHash: string
+    personId: string
+    date: string
+}
+
+export class BehavioralCounterRepository {
+    constructor(private cassandra: CassandraClient) {}
+
+    /**
+     * Truncates the behavioral_event_counters table (useful for tests)
+     */
+    async truncateCounters(): Promise<void> {
+        await this.cassandra.execute('TRUNCATE behavioral_event_counters')
```

Comment:
I'm generally not a fan of having these kind of things in the production code path. Truncate commands should only be in test files